jason-lab/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "06-tcp_servers",
+  "version": "1.0.0",
+  "description": "![DV](https://www.deltavcodeschool.com/wp-content/uploads/DeltaV.png) Lab 06: TCP Chat Server ======",
+  "main": "index.js",
```

Comment:
Make sure `main` aligns with your actual main file. `start` should usually be `node .` rather than the main filename.